Formula/percona-xtrabackup.rb

```diff
@@ -29,7 +29,7 @@ class PerconaXtrabackup < Formula
   depends_on "libfido2"
   depends_on "libgcrypt"
   depends_on "lz4"
-  depends_on "mysql-client"
+  depends_on "mysql"
```

Comment:
Do we still want this? `mysql-client` had the same fix as `mysql` applied recently.

Reply:
I have no strong feelings.  The tool (percona xtrabackup) is used for backing up mysql server, so I feel like having it tied to the server package is a bit more "correct" logically.

Reply:
> Thanks for the work you've put in so far! I cleaned up your branch a bit to get it ready to merge. I think we can do so as soon as CI is green.

Thanks! I've learned a lot about how HomeBrew works in the process.  Was fun.

Reply:
> I have no strong feelings. The tool (percona xtrabackup) is used for backing up mysql server, so I feel like having it tied to the server package is a bit more "correct" logically.

Ok, fine with keeping this as-is then.